app/assets/javascripts/discourse/app/templates/user-selector-autocomplete.hbr

```diff
@@ -2,9 +2,10 @@
   <ul>
     {{#each options.users as |user|}}
       <li>
-        <a href title="{{user.name}}">
+        <a href title="{{user.name}}" class="{{user.cssClasses}}">
           {{avatar user imageSize="tiny"}}
           <span class='username'>{{format-username user.username}}</span>
+          <span class='name'>{{user.name}}</span>
```

Comment:
This has the potential to not be included due to `SiteSetting.enable_names`:

https://github.com/discourse/discourse/blob/960ab4fdc4b666e0b52b66363263896663e38797/app/serializers/found_user_serializer.rb#L3-L9

So we just have to wrap this:

```handlebars
{{#if user.name}}
  <span class='name'>{{user.name}}</span>
{{/if}}
```